src/components/WixUiTpaConfig/WixUiTpaConfig.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+
+export interface WixUiTpaConfig {
+  mobile: boolean;
+}
+
+export interface WixUiTpaConfigProps {
+  mobile?: boolean;
+}
+
+const wixUiTpaConfig = React.createContext<WixUiTpaConfig>({
+  mobile: false,
+});
+
+export const WixUiTpaConfigProvider = wixUiTpaConfig.Provider;
+
+export const WixUiTpaConfigConsumer = wixUiTpaConfig.Consumer;
+
+export const withWixUiTpaConfig = <TProps extends object>(
+  WrappedComponent: React.FunctionComponent<TProps>,
+): React.FunctionComponent<TProps> =>
+  (props) => {
+    return (
+      <WixUiTpaConfigConsumer>
+        {(context: WixUiTpaConfig) => {
+          const deviceTypeProps = {
+            mobile: context.mobile,
+          };
+          return <WrappedComponent {...props} {...deviceTypeProps} />;
+        }}
+      </WixUiTpaConfigConsumer>
+    );
+  };
+
+export const WixUiTpaConfigWrapper = ({mobile = false}) => {
```

Comment:
This is the responsibility of the consumer, not the library.
if it's for test purposes, put it in the tests folder

Reply:
@shlomitc  in 'src/test/utils.tsx' ?

Reply:
yes, can be a good start